### PR TITLE
add contributing guidelines - fixes #167

### LINKED ISSUE #167: Contributing guidelines
Has there been any thought/discussion to setting up a contributing guidelines doc? I know we've all seen them, but here are a few examples I've tended to like:
- [npm](https://github.com/npm/npm/wiki/Contributing-Guidelines)
- [babel](https://github.com/babel/babel/blob/master/CONTRIBUTING.md)
- [Contributor Code of Conduct](http://contributor-covenant.org/version/1/0/0/)

I've always appreciated having the behavioral/philosophical guidelines set out, but I think it's also important that the basics of `git clone`, `npm test` etc. are there. Maybe we just link to some `contributing.md` doc in the FAQ at the bottom of the readme?

### PR DESCRIPTION
**Pull request by sindresorhus**
Fixes #167.

// @markthethomas @Whoaa512 @SamVerschueren

### CODE REVIEW

File: contributing.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+✨ Thanks for contributing to AVA! ✨
+
+## How can I contribute?
+
+### Improve documentation
+
+As a user of AVA you're the perfect candidate to help us improve our documentation. Typo corrections, error fixes, better explanations, more examples, etc. Open issues for things that could be improved. [Help translate our docs.](https://github.com/sindresorhus/ava-docs) Anything. Even improvements to this document.
+
+### Triage issues
```
Comment (vdemedes) [inline contributing.md]: Is there a simpler word to replace "triage"? Had no idea what it meant until now.

Perhaps "improve issues" or something along those lines?

Comment (sindresorhus) [inline contributing.md]: Good point. Fixed.

File: contributing.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+## How can I contribute?
+
+### Improve documentation
+
+As a user of AVA you're the perfect candidate to help us improve our documentation. Typo corrections, error fixes, better explanations, more examples, etc. Open issues for things that could be improved. [Help translate our docs.](https://github.com/sindresorhus/ava-docs) Anything. Even improvements to this document.
+
+### Improve issues
+
+Some issues are created with missing information, not reproducible, or plain invalid. Help make them easier to resolve. Triaging issues takes a lot of time that we could rather spend on fixing bugs and adding features.
```
Comment (vdemedes) [inline contributing.md]: "Triaging" still left over there in the 2nd sentence.

Comment (sindresorhus) [inline contributing.md]: Fixed

**Comment by vdemedes** [comment]
LGTM

**Comment by jamestalmage** [comment]
LGTM

File: contributing.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+- Search the issue tracker before opening an issue.
+- Ensure you're using the latest version of AVA.
+- Use a clear and descriptive title.
+- Include as much information as possible: Steps to reproduce the issue, error message, Node.js version, operating system, etc.
+- The more time you put into an issue, the more we will.
+- [The best issue report is a failing test proving it.](https://twitter.com/sindresorhus/status/579306280495357953)
+
+### Submitting a pull request
+
+- Non-trivial changes are often best discussed in an issue first.
```
Comment (SamVerschueren) [inline contributing.md]: Should we add a "why" to this? Something like `to prevent you from doing unnecessary work`.

Comment (sindresorhus) [inline contributing.md]: :+1:

**Pull request merged**